std/experimental/typecons.d

```diff
@@ -1024,3 +1024,10 @@ pure nothrow @safe unittest
     static assert(!__traits(compiles, arr ~= 4));
     assert((arr ~ 4) == [1, 2, 3, 4]);
 }
+
```

Comment:
Usually the link to the bug report is added as comment to the unittest, so that future developers know about this.